horace_core/sqw/page_operations/PageOp_sigvar_set.m

```diff
@@ -0,0 +1,34 @@
+classdef PageOp_sigvar_set < PageOpBase
+    % Single pixel page operation used by sigvar_set method
+    %
+
+    methods
+        function obj = PageOp_sigvar_set(varargin)
+            obj = obj@PageOpBase(varargin{:});
+            obj.op_name_ = 'sigvar_set';
+            obj.split_at_bin_edges = true;
+        end
+        function obj = init(obj,sqw_obj)
+            % sqw object here already have modified image so we use
+            % modified image to set pixels
```

Comment:
how do we know that the object has a modified image?

Reply:
This is the essence of the algorithm. We have sigvar (image) and setting pixels